buildtest/cli/__init__.py

```diff
@@ -353,8 +362,16 @@ def misc_menu(subparsers):
         "-y", "--yes", action="store_true", help="Confirm yes for all prompts"
     )
 
-    subparsers.add_parser("docs", help="Open buildtest docs in browser")
-    subparsers.add_parser("schemadocs", help="Open buildtest schema docs in browser")
+    # Subcommands that do not need to be shown in --help
+    if show_all_help:
+        subparsers.add_parser(
+            "schemadocs", help="Open buildtest schema docs in browser"
+        )
+        subparsers.add_parser("docs", help="Open buildtest docs in browser")
+    else:
+        subparsers.add_parser("schemadocs")
+        subparsers.add_parser("docs")
```

Comment:
can we avoid having redundant if/else statement instead just have one place perhaps at bottom where we add all the hidden parsers. 

You might just want to declare some variable for hidden parsers maybe here is one idea

```python
hidden_parser = { 'docs' : "Open buildtest docs in browser", 
                               'schemadocs': "..."
                             }
```

Then you could do the following

```python                             
for command, help_msg in hidden_parser.items():
   subparser.add_parser(command, help=help_msg)
```

If you want to capture aliases also then you will need to update the dictionary with more keys. This would require a bit refactoring.

Another question is do we need to do `subparsers.add_parser` in the `else` statement? I take it this is not required since the argparse will only need to build add hidden parsers when `-H` is specified otherwise it is not needed